### ISSUE TITLE
Small grammar error in error messages

### ISSUE THREAD

**Issue by msberends** [tidyverse/ggplot2#3602]
As error messages are parametrised, some end up having a grammatical error:

```r
theme(axis.line.x = "wrong!")
#> Error in (function (el, elname)  : 
#>   Element axis.line.x must be a element_line object.
```

Should be: *Element axis.line.x must be **an** element_line object*. This could be solved by checking the first character after the ' a ', although it's dependent on the sound, not the vowel itself, and it would go wrong e.g. here:

```r
theme(legend.spacing.x = "wrong!")
#> Error in (function (el, elname)  : 
#>   Element legend.spacing.x must be a unit object.
```
... which is of course already correct.

Maybe then only check for `^(a|e|i|o)`? Or perhaps change to:
```
Element {...} must be an object of type {...}.
```

**Comment by wendtke** [comment]
@clauswilke solved this issue with a commit ( f418e92bd8c249759e7af55625f562a845f53460 ) on November 15, 2019. 

The grammar should be correct, as long as the `{eldef$class}` object always beings with an element.

This issue can be closed now.

**Comment by clauswilke** [comment]
I'll reopen this. I think a better error message would say "must be an object of type ...". That resolves any possible future grammar issues.

**Comment by wendtke** [comment]
Addressed in PR #3775
